pkg/resource/certificate_authority/custom_delta.go

```diff
@@ -16,37 +16,30 @@ package certificate_authority
 import (
 	svcapitypes "github.com/aws-controllers-k8s/acmpca-controller/apis/v1alpha1"
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
-	"github.com/aws/aws-sdk-go/aws"
 )
 
 func customSetDefaults(
-	a *resource,
-	b *resource,
+	desired *resource,
+	latest *resource,
```

Comment:
I know in the `generator.yaml` there is a definition given with the parameters being a,b. Not sure how the generator works but wanted to call that out in case we need to fix `generator.yaml`